test/Microsoft.ML.FSharp.Tests/Microsoft.ML.FSharp.Tests.fsproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFramework></TargetFramework>
```

Comment:
is it supposed to have no value ?

Reply:
Yes.  I added `<TargetFramework>netcoreapp2.1` for all test projects.  But since this F# test wants to be multi-targeted (.NET Core and Desktop FX), you can't have `TargetFramework` set, or else you'd lose the multi-targetness.

---
In reply to: [211081608](https://github.com/dotnet/machinelearning/pull/690#discussion_r211081608) [](ancestors = 211081608)